### ISSUE TITLE
Does this work on a 2017 Macbook 14'1 running BigSur?

### ISSUE THREAD

**Issue by iCraft85** [kyoz/mac-arch#2]
Just wanted to know what version the docs work for because I didn't want to invest hours and hours and then realise it didn't work.

**Comment by kyoz** [comment]
Hi @iCraft85 ,

There is some issues on MacBook with T2 Sercure Chip, you can read more [here](https://discussions.apple.com/thread/251087440?answerId=252062188022#252062188022)

For 2017 Macbook, I think it'll work fine but I can't ensure anything. Just try it, if it not work, just delete the linux partition and everything will become like you haven't install anything.

This guide is pretty hardcore cause you have to install everything from scratch, so if you're not familiar with linux, I recommend you to install other linux distro like ubuntu or Manjaro...etc...This is a pretty detail [tutorial](https://dev.to/cmiranda/linux-on-macbook-pro-2016-1onb)

**Comment by kyoz** [comment]
BTW, if you want to have the same experience like i3 wm but on OSX, you can using [yabai](https://github.com/koekeishiya/yabai), it is more convenient and take less risk.

**Comment by iCraft85** [comment]
Ight thanks, I have tried yabai before its not bad but one of the big reasons I want to switch to Linux is because its so much more customisable, for example polybar and also compositors

**Comment by kyoz** [comment]
Yes, it's native linux so it run smoothly and fully customization. But it'll cost you sometime to fully install it on a MBP 😹

**Comment by mahashaman** [comment]
I have been using Archlinux since it first came out on my PC machine. I have an iMac 2012 with Catalina. I installed Ubuntu on this machine. But there was an overheating problem. Will there be such a problem if I install Arch? Or have you had such a problem?

**Comment by kyoz** [comment]
Yes, that is a known issue. But we also got [mbpfan](https://github.com/linux-on-mac/mbpfan) which allow to to control and throttle your fan. I'm always use it and doesn't have any overheat issue so far.

**Comment by jonaskaira** [comment]
I did everything right. Just on OSX I got this error on the bless command:

sudo bless --device /dev/disk0s5 --setBoot
Could not set boot device property: 0xe00002e2

Finally, the GRUB screen appears. It expects me to enter a GRUB command.

**Comment by kyoz** [comment]
@jonaskaira Can you try [disable SIP](https://developer.apple.com/documentation/security/disabling_and_enabling_system_integrity_protection?language=objc)

BTW What OSX version are you using?

**Comment by jonaskaira** [comment]
Yes. I disabled SIP with csrutil. My system is 2012 iMac Catalina 10.15.7.
Now, i have a much bigger problem.

I thought that I could install Ubuntu and Arch together and introduce Arch into the bootloader that Ubuntu uses.
I used Disk Utility to reorganize the partitions I had allocated, and everything seemed fine. I rebooted the system. Surprise. The Macos partition was gone. In Disk Utility there was an unknown partition called disk0s2. For 3-4 hours I tried every way described on the web. The result was frustrating. Now, I'm left with Mountain Lion's Safari.

Ps: Bless was work.
